projectfiles/visualstudio-2010/fba_vs2010.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">

```

Comment:
Does this change retain compatibility with the older MSVC?  (re: ToolsVersion="15.0")

Reply:
It should do, I tested it with vs2017 and vs2015 tfs as well as the latest sharpdevelop and it worked ok. However, the caveat there is that I have the latest Windows platform sdk installed which might be what this is referring too. Easily changed if someone is using older VS or wants to use the older tools (this should be the only change to the project file that would stop it working)